php/boot-fs.php

```diff
@@ -3,7 +3,7 @@
 /**
  * This file needs to parse without error in PHP < 5.3
  */
-if(PHP_SAPI != 'cli') {
+if(PHP_SAPI != 'cli' && PHP_SAPI != 'phpdbg') {
```

Comment:
Can we also rewrite this logic to be `if(!in_array(PHP_SAPI, array('cli', 'phpdbg'))) { exit; }` so it isn't a brainfuck. Also, it's much more clear that it's backwards compatible if phpdbg isn't present.